Source/Charts/Utils/ViewPortHandler.swift

```diff
@@ -434,7 +434,7 @@ open class ViewPortHandler: NSObject
     
     @objc open func isInBoundsTop(_ y: CGFloat) -> Bool
     {
-        return _contentRect.origin.y <= y
+        return _contentRect.origin.y <= y + 1.0
```

Comment:
Why?

Reply:
when iOS9 ，_contentRect.origin.y is 30.0，y is 2.999999986，so The y value error is plus or minus 1

Reply:
I don't get it. what's the problem again?